Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -34,9 +35,24 @@ namespace Multiplayer
         m_networkEditorInterface = AZ::Interface<INetworking>::Get()->CreateNetworkInterface(
             AZ::Name(MpEditorInterfaceName), ProtocolType::Tcp, TrustZone::ExternalClientToServer, *this);
         m_networkEditorInterface->SetTimeoutMs(AZ::TimeMs{ 0 }); // Disable timeouts on this network interface
-        ActivateDedicatedEditorServer();
+        CrySystemEventBus::Handler::BusConnect();
```

Comment:
Any chance CrySystem is already initialized at this point?

Reply:
Good question! 
For Game/Server Launchers the system components get turned on during Launcher.cpp Run() method where is says gameApplication.Start(). . . below that logic is where the CrySystem is created.

For Editor the magic happens in CryEditMain: here the system components are created deep inside AZToolsApp.Start() and below that there's the call to crysystem init deep inside CCryEditApp::instance()->InitInstance

In a perfect world, the logging system would have instead been attached to a system component which my multiplayer system component would have easily been able to require service.

Reply:
@AMZN-stankowi did some work with the life cycle of the application recently that signals the application events for handlers to listen for

Reply:
Nice! I'll follow up with @AMZN-stankowi

Reply:
Second this, if we can avoid relying on CrySystemEventBus that'd be great

Reply:
Fixed! Using the new ComponentApplicationLifecycle